stylesheets/commons/Brackets.scss

```diff
@@ -311,6 +311,14 @@
 	color: #c04845;
 }
 
+.brkts-identityv-score-color-survivor {
+	color: #3ab4e8;
+}
+
+.brkts-identityv-score-color-hunter {
+	color: #b81414;
+}
+
```

Comment:
@Kanopedia @Hesketh2 could you check if these colors are okay?
i fetched them from
- https://liquipedia.net/commons/images/thumb/e/e0/IDV_Survivor_icon_allmode.png/16px-IDV_Survivor_icon_allmode.png
- https://liquipedia.net/commons/images/thumb/9/96/IDV_Hunter_icon_allmode.png/16px-IDV_Hunter_icon_allmode.png